app/articles/server/methods/admin.js

```diff
@@ -0,0 +1,87 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { API } from '../utils/url';
+import { settings } from '../../../settings';
+
+const api = new API();
+
+function setupGhost(user, token, rcUrl) {
+	const blogTitle = settings.get('Article_Site_title');
+	const data = {
+		setup: [{
+			rc_url: rcUrl,
+			rc_id: user._id,
+			rc_token: token,
+			name: user.name,
+			email: user.emails[0].address,
```

Comment:
Is this the best approach to get email? Is user.emails[0] always verified?

Reply:
We can fetch the verified mail. There may be a case that the user doesn't have any verified mail. Then should I just use this?